README.md

```diff
@@ -5,12 +5,22 @@
 ### Overview
 You can use the `cloud-object-storage` template to deploy IBM Cloud Functions assets for you.  The assets created by this template are described in the `manifest.yaml` file, which can be found at `template-cloud-object-storage/runtimes/nodejs/manifest.yaml`.
 
+The template creates a form where users can upload a photo into a cloud object storage bucket.  The template deploys a Web Action, *initialHtml*, that contains an html form, which passes its data to the *handleForm* Action.  When the form data is parsed by the *handleForm* Action, the image uploaded in the form is stored in the Cloud Object Storage instance.  Because this is a public web action, keep in mind that any users can post photos into your bucket.  This web action could be [secured](https://console.bluemix.net/docs/openwhisk/openwhisk_webactions.html#securing-web-actions).
```

Comment:
The Cloud Object Storage template creates a form where users can upload a photo into a Cloud Object Storage bucket. The template deploys a Web action called *initialHtml*, that contains an `HTML` form, which passes its data to the *handleForm* Action. When the form data is parsed by the *handleForm* Action, the image uploaded in the form is stored in the Cloud Object Storage instance.  Since this is a public Web action, be aware that _any_ user can post photos into your bucket. For added security, you can [secure](https://console.bluemix.net/docs/openwhisk/openwhisk_webactions.html#securing-web-actions) the Web action.